index.html

```diff
@@ -1,114 +1,223 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charSet="utf-8"/>
-    <meta http-equiv="x-ua-compatible" content="ie=edge"/>
-    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"/>
+
+<head>
+    <meta charSet="utf-8" />
+    <meta http-equiv="x-ua-compatible" content="ie=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no" />
     <title>is-a.dev | Free sub-domains for developers</title>
+    <link rel="stylesheet" href="https://a.thered.sh/assets/fonts/Roboto/font.css">
+    <link rel="stylesheet" href="https://a.thered.sh/assets/fonts/UbuntuMono/font.css">
+    <link rel="stylesheet" href="https://a.thered.sh/assets/fa/css/all.min.css">
```

Comment:
It'll be better if all font and css files were served from this repo instead.

Reply:
It should be served from / instead for now, my repo has both fa and fonts there